test/TestServerProjectsLowLevel/security-aad/AutorestSecurityAadClient.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace security_aad_LowLevel
+{
+    [CodeGenSuppress("AutorestSecurityAadClient", typeof(TokenCredential), typeof(Uri), typeof(AutorestSecurityAadClientOptions))]
+    public partial class AutorestSecurityAadClient
+    {
+        /// <summary> Initializes a new instance of AutorestSecurityAadClient. </summary>
+        /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
+        /// <param name="endpoint"> server parameter. </param>
+        /// <param name="options"> The options for configuring the client. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="credential"/> or <paramref name="endpoint"/> is null. </exception>
+        public AutorestSecurityAadClient(TokenCredential credential, Uri endpoint, AutorestSecurityAadClientOptions options)
+        {
+            Argument.AssertNotNull(credential, nameof(credential));
+            Argument.AssertNotNull(endpoint, nameof(endpoint));
+            options ??= new AutorestSecurityAadClientOptions();
+
+            ClientDiagnostics = new ClientDiagnostics(options, true);
+            _tokenCredential = credential;
+            _pipeline = HttpPipelineBuilder.Build(options, Array.Empty<HttpPipelinePolicy>(), new HttpPipelinePolicy[] { new MockBearerTokenAuthenticationPolicy(_tokenCredential, AuthorizationScopes) }, new ResponseClassifier());
+            _endpoint = endpoint;
+        }
+
+        // Only for bypassing HTTPS check purpose
+        public class MockBearerTokenAuthenticationPolicy : BearerTokenAuthenticationPolicy
```

Comment:
Can we move this out of the generated codes. And inject this mock policy when initializing the client? Thus we don't need to maintain `[CodeGenSuppress]` which is easy to loose synchronization with latest autorest changes.